src/content/1.7/development/page-reference/back-office/product/form/old-product-form.md

```diff
@@ -0,0 +1,198 @@
+---
+title: The old Product Form
+menuTitle: Old Form
+weight: 1
+---
+
+# The Old Product Form
+{{< minver v="1.7.0" title="true" >}}
```

Comment:
This is only needed when something is introduced in a minor (or patch) version. The old form was introduced in 1.7.0 so not needed here. 

You should clarify at the top of the other page that the form is being remade for 1.7.8 to be coherent with the rest of the migration and that for versions 1.7.0~1.7.7 people should read this article.

That said, maybe this page should be called something like "Product form before 1.7.8".

Reply:
Good idea, done!